### ISSUE TITLE
Unexpected token export

### ISSUE THREAD

**Issue by epodivilov** [alfredosalzillo/rollup-plugin-multi-input#9]
After updating the plugin to the latest version, I began to receive the following error. 
```
/Project/node_modules/lodash-es/fromPairs.js:28
export default fromPairs;
^^^^^^

SyntaxError: Unexpected token export
    at Module._compile (internal/modules/cjs/loader.js:718:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Object.require.extensions..js (/Project/node_modules/rollup/bin/rollup:1039:17)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Project/node_modules/rollup-plugin-multi-input/dist/plugin.js:38:41)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Module._extensions..js (internal/modules/cjs/loader.js:785:10)
```

On version 1.0.0 everything works fine.

OS: MacOS Mojave 10.14.5
Node: 12.3.1
Rollup: 1.16.2

**Comment by alfredosalzillo** [comment]
Thanks for the report. Can you share your rollup configuration?

**Comment by epodivilov** [comment]
```
import multiInput from "rollup-plugin-multi-input";
import reactSvg from "rollup-plugin-react-svg";

export default {
  input: ["src/**/*.js"],
  output: {
    dir: 'rollup',
    format: 'cjs'
  },
  external: ['react'],
  plugins: [
    multiInput(),
    reactSvg({
      svgo: {
        multipass: true,
        plugins: [
          { removeXMLNS: true },
          { removeTitle: true },
          { removeXMLProcInst: true },
          { removeDesc: true },
          { removeDoctype: true },
          { collapseGroups: true },
          { removeComments: true },
          { removeAttrs: { attrs: '(fill-rule|style)' } },
        ]
      }
    })
  ]
};
```

**Comment by alfredosalzillo** [comment]
Should be fixed with the 1.0.2
